tests/data/aws/ecr.py

```diff
@@ -26,47 +26,88 @@
         },
     ],
 }
-
+DESCRIBE_IMAGES = {
```

Comment:
Adding this to have sample data from [boto3 ec3.describe_images](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ecr/client/describe_images.html).